ios/FluentUI.Demo/FluentUI.Demo/Demos/SideTabBarDemoController.swift

```diff
@@ -327,7 +327,7 @@ extension SideTabBarDemoController: UITableViewDataSource {
 // MARK: - SideTabBarDemoController: DemoAppearanceDelegate
 extension SideTabBarDemoController: DemoAppearanceDelegate {
     func themeWideOverrideDidChange(isOverrideEnabled: Bool) {
-        guard let fluentTheme = self.view.window?.fluentTheme else {
+        guard let fluentTheme = contentViewController?.view.window?.fluentTheme else {
```

Comment:
self.view.window is actually different from contentviewcontroller.view.window?

Reply:
It absolutely is! self.view.window is nil.

Reply:
why is that?

Reply:
huh.... line 136 is interesting way we have this sidetabbardemocontroller viewcontroller hierachy setup.

Reply:
Yes you beat me to it :) the contentViewController is what is presenting the SideTabBar control, not self.

Reply:
can you make sure you put "why" this change fixes the bug in your PR description ,please?